Adyen/Model/Checkout/PayPalDetails.cs

```diff
@@ -0,0 +1,81 @@
+﻿#region Licence
+// 
+//                        ######
+//                        ######
+//  ############    ####( ######  #####. ######  ############   ############
+//  #############  #####( ######  #####. ######  #############  #############
+//         ######  #####( ######  #####. ######  #####  ######  #####  ######
+//  ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+//  ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  #############  #############  #############  #############  #####  ######
+//   ############   ############  #############   ############  #####  ######
+//                                       ######
+//                                #############
+//                                ############
+// 
+//  Adyen Dotnet API Library
+// 
+//  Copyright (c) 2020 Adyen B.V.
+//  This file is open source and available under the MIT license.
+//  See the LICENSE file for more info.
+#endregion
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Adyen.Model.Checkout
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    [DataContract]
+    public class PayPalDetails : IOneOfPaymentRequestPaymentMethod
+    {
+        [JsonConverter(typeof(StringEnumConverter))]
+        public enum SubtypeEnum
+        {
+            /// <summary>
+            /// Enum SDK for value: SDK
+            /// </summary>
+            [EnumMember(Value = "sdk")]
+            SDK = 1,
+
+            /// <summary>
+            /// Enum SDK for value: SDK
+            /// </summary>
+            [EnumMember(Value = "redirect")]
+            Redirect = 1,
+        }
+
+        /// <summary>
+        /// PayPal subtype
+        /// </summary>
+        /// <value>enum subtype</value>
+        [DataMember(Name = "subtype", EmitDefaultValue = false)]
+        [JsonProperty(PropertyName = "subtype")]
+        public SubtypeEnum Subtype;
+
+        /// <summary>
+        /// **paypal**
+        /// </summary>
+        /// <value>**paypal**</value>
+        [DataMember(Name = "type", EmitDefaultValue = false)]
+        [JsonProperty(PropertyName = "type")]
+        public string Type { get; set; } = "paypal";
```

Comment:
can we make from paypal an constant ? see how we do it in java: https://github.com/Adyen/adyen-java-api-library/blob/develop/src/main/java/com/adyen/model/checkout/details/PayPalDetails.java#L43